deepwell/src/api.rs

```diff
@@ -166,6 +166,14 @@ fn build_routes(mut app: ApiServer) -> ApiServer {
         .get(site_custom_domain_retrieve);
     app.at("/site/fromDomain/:domain").get(site_get_from_domain);
 
+    // Site Membership
+    app.at("/site/members").get(membership_site_retrieve);
+    app.at("/site/members/get").get(membership_retrieve);
+    app.at("/site/members/add").put(membership_put);
+    app.at("/site/members/remove").delete(membership_delete);
```

Comment:
The root here should be `/site/member`, singular.

Reply:
These route/methods are not correct. The only reason `/get` is a separate thing here is because in HTTP you cannot submit a body with a GET request, so that is `PUT /foo/get` in order to emulate a "GET with a body" (PUT can have a body and is idempotent so that's why it's used here).

The others should not have separate routes. It should be `GET /site/member`, `PUT /site/member`, `DELETE /site/member`.

Reply:
First time for everything 😅
will fix

Reply:
No worries

Reply:
Also site membership retrieve is its own thing and also takes a body, make that `PUT /site/member/site/get` or something